llvm/include/llvm/CodeGen/TargetPassConfig.h

```diff
@@ -140,6 +140,9 @@ class TargetPassConfig : public ImmutablePass {
   /// callers.
   bool RequireCodeGenSCCOrder = false;
 
+  /// Enable LoopTailFold immediately after LSR
+  bool EnableLoopTailFold = false;
```

Comment:
```suggestion
  /// Enable LoopTermFold immediately after LSR
  bool EnableLoopTermFold = false;
```
I assume you meant this?